api/token/token.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2017 clair authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package token
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"time"
+
+	fernet "github.com/fernet/fernet-go"
```

Comment:
Do you actually need this? I'm pretty sure without it, it still imports as `fernet`.